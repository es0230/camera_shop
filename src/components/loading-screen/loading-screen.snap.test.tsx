/**
 * @jest-environment jsdom
 */

import { render } from '@testing-library/react';
import LoadingScreen from './loading-screen';

describe('Testing Footer component', () => {
  it('should render correctly', () => {
    const { container } = render(<LoadingScreen />);

    expect(container).toMatchSnapshot();
  });
});
