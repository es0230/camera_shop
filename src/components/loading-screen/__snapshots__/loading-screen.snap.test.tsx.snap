// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Footer component should render correctly 1`] = `
<div>
  <h1>
    Загрузка...
  </h1>
</div>
`;
